libusb/os/windows_common.c

```diff
@@ -279,6 +279,7 @@ enum libusb_transfer_status usbd_status_to_libusb_transfer_status(USBD_STATUS st
 	case USBD_STATUS_CANCELED:
 		return LIBUSB_TRANSFER_CANCELLED;
 	case USBD_STATUS_ENDPOINT_HALTED:
+	case USBD_STATUS_STALL_PID:
```

Comment:
Just wondering what is the reason to add this one? Any potential side-effect?

Reply:
https://github.com/libusb/libusb/pull/1257#issuecomment-1451687864

no side-effect i think, this changes follows the usb specification.